main/core/src/util/Compat.scala

```diff
@@ -0,0 +1,13 @@
+package mill.util
+
+import scala.reflect.macros.blackbox.Context
+
+object Compat{
```

Comment:
This was removed from Ammonite there https://github.com/lihaoyi/Ammonite/pull/981/files